src/hooks/useSubscriptions.js

```diff
@@ -1,9 +1,33 @@
 import { useCallback, useEffect, useRef, useState } from 'react'
 import {
+  transformConfigData,
   transformProposalData,
   transformStakeHistoryData,
 } from '../lib/data-utils'
 
+export function useConfigSubscription(convictionVoting) {
+  const [config, setConfig] = useState([])
```

Comment:
should be `null` right?

```suggestion
  const [config, setConfig] = useState(null)
```